modules/blocks/chain.js

```diff
@@ -422,20 +379,7 @@ Chain.prototype.applyBlock = function(block, saveBlock, cb) {
 				new Promise((resolve, reject) => {
 					modules.accounts.getAccount(
 						{ publicKey: transaction.senderPublicKey },
-						(err, sender) => {
-							if (err) {
-								// Fatal error, memory tables will be inconsistent
-								err = [
-									'Failed to apply transaction:',
-									transaction.id,
-									'-',
-									err,
-								].join(' ');
-								library.logger.error(err);
-								library.logger.error('Transaction', transaction);
-
-								reject(err);
-							}
+						(accountErr, sender) => {
 							// DATABASE: write
 							modules.transactions.apply(
```

Comment:
Same as the comment above - no reason to perform the `apply` operation if accountErr happened.

Reply:
Yeah, sure.